playbook/roles/mongodb/README.md

```diff
@@ -0,0 +1,45 @@
+Ansible Role: Mongodb
+=========
+
+Installs [Mongodb](https://www.mongodb.com) on RHEL/CentOS or Debian/Ubuntu.
+
+[![Build Status](https://travis-ci.org/d4rkstar/ansible-role-mongodb.svg?branch=master)](https://travis-ci.org/d4rkstar/ansible-role-mongodb)
```

Comment:
Build status should be removed from the README.md. We have one top level build status that should be enoug :D